### PR TITLE
Feat/ints 33/progress bar

### PR DESCRIPTION
**Pull request by IkkaJabua**

### CODE REVIEW

File: app/components/Button/HeartShapeBtn.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+interface Props {
+  isActive: boolean;
+  isDisabled: boolean;
+  onClick: () => void;
+}
+
+export default function HeartShapeBtn ({ isActive, isDisabled, onClick }: Props) {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const getIconSource = () => {
```
Comment (1Lazare) [inline app/components/Button/HeartShapeBtn.tsx]: delete this fucntion you dont need

File: app/components/Button/HeartShapeBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+  };
+
+  return (
+    <button
+      className={style.btn}
+      onClick={handleClick}
+      disabled={isDisabled}
+    >
+      <Image
```
Comment (1Lazare) [inline app/components/Button/HeartShapeBtn.tsx]: use Icon Component

File: app/components/MusicPlayer/Contorls.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    onVolumeChange,
+    volume,
+    isLooping,
+    onToggleLoop,
+    isShuffling,
+    onToggleShuffle,
+    currentTime,
+    duration,
+    onTimeChange,
+}: Props) => {
```
Comment (1Lazare) [inline app/components/MusicPlayer/Contorls.tsx]: refactor this to one prop

File: app/components/MusicPlayer/Contorls.tsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    volume: number;
+    isLooping: boolean;
+    onToggleLoop: () => void;
+    isShuffling: boolean;
+    onToggleShuffle: () => void;
+    currentTime: number;
+    duration: number;
+    onTimeChange: (newTime: number) => void;
+    backgroundImage: string
+}
```
Comment (1Lazare) [inline app/components/MusicPlayer/Contorls.tsx]: create for this new file and write this interface here

**Comment by IkkaJabua** [comment]
done

**Review by 1Lazare — APPROVED** [review]

**Pull request merged**